src/alfasim_sdk/result_reader/aggregator.py

```diff
@@ -204,22 +220,18 @@ def from_dict(cls, data: Dict[str, Any]) -> Self:
                 sample_indexes=data["sample_indexes"],
             )
 
-    items: Dict[str, UPItem] = attr.ib(validator=attr.validators.instance_of(Dict))
-    result_directory: Path = attr.ib(validator=attr.validators.instance_of(Path))
-
-    @classmethod
-    def empty(cls, result_directory: Path) -> Self:
-        return UncertaintyPropagationAnalysesMetaData(
-            items={}, result_directory=result_directory
-        )
```

Comment:
A note for reviewers: this was removed there should be no case in which we have empty metadata for this type of results, once they don't have the concept of partial completion. The presence of these empty objects could lead to confusion and ultimately bugs, so they were ripped of.